test/integration/smart_answer_flows/pip_checker_test.rb

```diff
@@ -55,29 +55,29 @@ class PIPCheckerTest < ActiveSupport::TestCase
   context "getting DLA" do
     setup do
       add_response 'yes'
-      Timecop.travel('2014-02-03')
+      Timecop.travel('2013-02-03')
```

Comment:
How can we be sure that the time set by `Timecop` is reset after each test?
Shouldn't we have a `teardown` section?